app/components/add-reimbursement/component.js

```diff
@@ -85,7 +86,22 @@ export default class AddReimbursementComponent extends Component {
   }
 
   @action
-  updateContributor(event) {
+  async addExpensesFromFile (evt) {
```

Comment:
Thanks! I added a note with a link to your comment for now, since this is kind of an undocumented and hidden feature for someone who knows what they're doing as of now. Mostly just added it so I could easily migrate the existing data, even though it may be useful for other imports in the future. It's basically instant and you wouldn't navigate away while adding a reimbursement, so I think it's OK to leave it as is for now and spend time on more important things.